resources/watch.rb

```diff
@@ -0,0 +1,49 @@
+default_action(:create)
```

Comment:
NIT: Here too remove parantheses, or use parenthenses in other files, for consistency